packages/governance/Scarb.toml

```diff
@@ -25,6 +25,8 @@ fmt.workspace = true
 starknet.workspace = true
 openzeppelin_access = { path = "../access" }
 openzeppelin_introspection = { path = "../introspection" }
+openzeppelin_account = { path = "../account"}
+openzeppelin_token = { path= "../token"}
```

Comment:
```suggestion
openzeppelin_account = { path = "../account" }
openzeppelin_token = { path= "../token" }
```

Reply:
Can I get `scarb fmt` to do this for me? 😅